docs/admin_api/user_admin_api.rst

```diff
@@ -1,3 +1,5 @@
+These commands will require an ``access_token`` of a server admin. Make sure you read the ``admin_api`` documentation on how to obtain and use such an ``access_token`` (section `"Using an admin token" <https://github.com/matrix-org/synapse/tree/master/docs/admin_api#using-an-admin-access_token>`_) before going any further.
```

Comment:
this is true of all the admin apis; I don't think it's helpful just to add it to this file.

Also: please wrap your markdown at 80 characters.

Reply:
@axelsimon please can you put your comments on the thread that they refer to.

In reply to https://github.com/matrix-org/synapse/pull/6581#issuecomment-567964653:

I guess you're right. This is a lot of words though. I think it would be simpler and clearer to change the lines that say

```
including an ``access_token`` of a server admin.
```

to include a link:

```
including an ``access_token`` of a server admin (see `README.rst`_).
```